package.json

```diff
@@ -8,22 +8,22 @@
     "url": "https://github.com/yahoo/fluxible"
   },
   "scripts": {
-    "build": "lerna bootstrap --hoist --since --include-filtered-dependencies",
-    "cover": "lerna run cover --since",
+    "build": "lerna bootstrap --hoist --since 815a09f --include-filtered-dependencies --no-ci",
```

Comment:
`815a09f` is a hack :(  It is the sha for fluxible-router-1.x branch.  For some reason `--since origin/fluxible-router-1.x` does not work in travis.  Will need to figure out what is going on later.